src/log/log_unix.go

```diff
@@ -11,6 +11,7 @@
 // either express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
+//go:build darwin || freebsd || linux || netbsd || openbsd
```

Comment:
I think `go:build` isn't added [until 1.17](https://go.dev/doc/go1.17#gofmt).

Any reason not to upgrade to latest go version (which also removes the `+build` lines)?